website/docs/r/globalaccelerator_endpoint_group.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "aws"
+page_title: "AWS: globalaccelerator_endpoint_group"
+sidebar_current: "docs-aws-resource-globalaccelerator-endpoint-group"
+description: |-
+  Provides a Global Accelerator endpoint group.
+---
+
+# Resource: globalaccelerator_endpoint_group
+
+Provides a Global Accelerator endpoint group.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The Amazon Resource Name (ARN) of the accelerator.
+
+## Example Usage
+
+```hcl
+resource "globalaccelerator_endpoint_group" "example" {
+  listener_arn            = "arn:aws:globalaccelerator::123456789012:accelerator/1234abcd-abcd-1234-abcd-1234abcdefgh/listener/0123vxyz"
+  endpoint_configuration {
+    EndpointId   = "arn:aws:elasticloadbalancing:us-east-1:123456789012:loadbalancer/app/my-load-balancer/50dc6c495c0c9188"
+    Weight = "100"
```

Comment:
The argument name here is lowercase `weight` and is an integer which does not need to be wrapped in quotes:

```suggestion
    weight = 100
```